examples/plain-javascript/README.md

```diff
@@ -30,10 +30,13 @@ Then, you'll get this:
 .
 ├── cldrjs
 │   ├── cldr.js
+│   ├── node_main.js
 │   └── cldr
 │       ├── event.js
 │       └── supplemental.js
-└── index.html
+│       └── unresolved.js
```

Comment:
`unresolved.js` isn't used by this example, therefore I think it shouldn't be listed here (for simplicity).